assets/scss/_base.scss

```diff
@@ -902,6 +902,12 @@ section#cncf {
   margin: 0;
 }
 
+//Table Content
+.tab-content table{
+  table-layout:fixed;
+  width:100%;
+}
```

Comment:
Does this affect other tables as well?

Reply:
> Does it affect other tables as well?

No, it doesn't affect other tables.I have checked the whole website, and there is only one table under the tabs in this section. Even if it is added in future, it will not affect the layout.